runtime/autoload/man.vim

```diff
@@ -2,15 +2,21 @@
 
 let s:man_find_arg = "-w"
 
-" TODO(nhooyr) Completion may work on SunOS; I'm not sure if `man -l` displays
-" the list of searched directories.
-try
-  if !has('win32') && $OSTYPE !~? 'cygwin\|linux' && system('uname -s') =~? 'SunOS' && system('uname -r') =~# '^5'
-    let s:man_find_arg = '-l'
-  endif
-catch /E145:/
-  " Ignore the error in restricted mode
-endtry
+function s:init() abort
+  " TODO(nhooyr) Completion may work on SunOS; I'm not sure if `man -l` displays
+  " the list of searched directories.
+  try
+    if !has('win32') && $OSTYPE !~? 'cygwin\|linux' && system('uname -s') =~? 'SunOS' && system('uname -r') =~# '^5'
+      let s:man_find_arg = '-l'
+    endif
+    let s:localfile_arg = 1
+    call s:get_page(s:get_path('', 'man'))
```

Comment:
get_page result has NL char at the end, which needs to be chopped off (see other usage in this file). Yucky, but oh well. This is the last bug fix man.vim will ever need, right? :)